dle/entrypoint

```diff
@@ -3,6 +3,8 @@
 set -o errexit
 set -o pipefail
 
+sed "s/API_ENDPOINT/$API_ENDPOINT/g" -i search/static/search/es_search.js
```

Comment:
Check out how `ELASTIC_HOST` is set. In `search/views.py`, we are passing the `ELASTIC_HOST` var to the template which in turn creates a global Javascript var. `ELASTIC_HOST` is currently the reverse("api:searchkit_root") URL. You can do the same thing with the `API_ENDPOINT` instead of hacking it in with sed. More maintainable to have one style of doing things and I don't think modifying files during entrypoint startup is best practice.

Reply:
Not sure I agree that a better solution in this case. Doing a find and replace is fine.

Reply:
It means that we're handling this single case differently than every other environment variable. Inconsistency is harder to maintain. But it works so whatever, your call.

This should include the URL protocol and port, as this should soon be HTTPS with no port for prod.